src/Domain/OutputType.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace PhpTuf\ComposerStager\Domain;
+
+/**
+ * Process output type constants.
+ *
+ * @see \Symfony\Component\Process\Process::readPipes
+ */
+final class OutputType
```

Comment:
Why is this class needed?

Reply:
I don't want clients depending directly on third party code, i.e., Symfony. You could consider it a really lightweight instance of an anti-corruption layer.